### ISSUE TITLE
InternalUtils.symbol: tree is null

### ISSUE THREAD

**Issue by cushon** [typetools/checker-framework#3888]
```java
import org.checkerframework.checker.nullness.qual.PolyNull;

abstract class T {

  interface L<X> {}

  interface E {}

  public interface F<V, Y> {
    Y a(V v);
  }

  abstract <T> void f(F<T, Boolean> f);

  void c(F<E, @PolyNull L> o) {
    f((E vm) -> o.a(vm) == null);
  }
}
```

```
./checker-framework-3.7.1/checker/bin/javac -processor Nullness T.java
error: InternalUtils.symbol: tree is null
  ; The Checker Framework crashed.  Please report the crash.
  Compilation unit: T.java
  Last visited tree at line 3 column 1:
  abstract class T {
  Exception: java.lang.Throwable; java.lang.Throwable
        at org.checkerframework.javacutil.BugInCF.<init>(BugInCF.java:16)
        at org.checkerframework.javacutil.TreeUtils.elementFromTree(TreeUtils.java:459)
        at org.checkerframework.javacutil.TreeUtils.elementFromDeclaration(TreeUtils.java:517)
        at org.checkerframework.checker.nullness.NullnessTransfer.strengthenAnnotationOfEqualTo(NullnessTransfer.java:206)
        at org.checkerframework.checker.nullness.NullnessTransfer.strengthenAnnotationOfEqualTo(NullnessTransfer.java:62)
        at org.checkerframework.framework.flow.CFAbstractTransfer.visitEqualTo(CFAbstractTransfer.java:744)
        at org.checkerframework.framework.flow.CFAbstractTransfer.visitEqualTo(CFAbstractTransfer.java:94)
        at org.checkerframework.dataflow.cfg.node.EqualToNode.accept(EqualToNode.java:31)
        at org.checkerframework.dataflow.analysis.AbstractAnalysis.callTransferFunction(AbstractAnalysis.java:338)
        at org.checkerframework.dataflow.analysis.ForwardAnalysisImpl.callTransferFunction(ForwardAnalysisImpl.java:390)
        at org.checkerframework.dataflow.analysis.ForwardAnalysisImpl.performAnalysisBlock(ForwardAnalysisImpl.java:130)
        at org.checkerframework.dataflow.analysis.ForwardAnalysisImpl.performAnalysis(ForwardAnalysisImpl.java:107)
        at org.checkerframework.framework.flow.CFAbstractAnalysis.performAnalysis(CFAbstractAnalysis.java:109)
        at org.checkerframework.framework.type.GenericAnnotatedTypeFactory.analyze(GenericAnnotatedTypeFactory.java:1341)
        at org.checkerframework.framework.type.GenericAnnotatedTypeFactory.performFlowAnalysis(GenericAnnotatedTypeFactory.java:1267)
        at org.checkerframework.framework.type.GenericAnnotatedTypeFactory.checkAndPerformFlowAnalysis(GenericAnnotatedTypeFactory.java:1686)
        at org.checkerframework.framework.type.GenericAnnotatedTypeFactory.preProcessClassTree(GenericAnnotatedTypeFactory.java:343)
        at org.checkerframework.common.basetype.BaseTypeVisitor.visitClass(BaseTypeVisitor.java:322)
        at org.checkerframework.common.basetype.BaseTypeVisitor.visitClass(BaseTypeVisitor.java:175)
        at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCClassDecl.accept(JCTree.java:808)
        at jdk.compiler/com.sun.source.util.TreePathScanner.scan(TreePathScanner.java:56)
        at org.checkerframework.framework.source.SourceVisitor.visit(SourceVisitor.java:82)
        at org.checkerframework.framework.source.SourceChecker.typeProcess(SourceChecker.java:916)
        at org.checkerframework.common.basetype.BaseTypeChecker.typeProcess(BaseTypeChecker.java:503)
        at org.checkerframework.javacutil.AbstractTypeProcessor$AttributionTaskListener.finished(AbstractTypeProcessor.java:190)
        at jdk.compiler/com.sun.tools.javac.api.ClientCodeWrapper$WrappedTaskListener.finished(ClientCodeWrapper.java:828)
        at jdk.compiler/com.sun.tools.javac.api.MultiTaskListener.finished(MultiTaskListener.java:132)
        at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.flow(JavaCompiler.java:1394)
        at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.flow(JavaCompiler.java:1351)
        at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:947)
        at jdk.compiler/com.sun.tools.javac.main.Main.compile(Main.java:311)
        at jdk.compiler/com.sun.tools.javac.main.Main.compile(Main.java:170)
        at jdk.compiler/com.sun.tools.javac.Main.compile(Main.java:57)
        at jdk.compiler/com.sun.tools.javac.Main.main(Main.java:43)
```

**Comment by cushon** [comment]
On preview this looks like a dupe of https://github.com/typetools/checker-framework/issues/3850, sorry for the noise.

**Comment by mernst** [comment]
@cushon No problem.  I confirmed that the fix for #3850 also fixes this, and I'll add your test case to our test suite.
